Replace override for layout_builder_st and toc_api with D11 contrib versions.

olstjos opened the pull request:
Resolve release blocker(s)

olstjos committed:
Replace override for layout_builder_st and toc_api with D11 compatible contrib version

Resolve release blocker(s)

smulvih2 commented:
@olstjos thanks for this, this fixes the remaining blockers for D11 WxT! I can test later tonight and merge into the 6.1.x

olstjos committed:
Overrides replacement

Fix for toc_filter + D11 , layout_builder_st replace override and toc_api replace override.

olstjos committed:
Update composer.json - security advisory update for block_class, require at least 4.0.1

olstjos committed:
Update composer.json - The good patch

olstjos committed:
Update composer.json, layout_builder_st is now working

olstjos committed:
Update composer.json - preserve patch we were using ipreviously
